### ISSUE TITLE
Updating and contributing

### ISSUE THREAD

**Issue by aboutofpluto** [davidmz/apng-js#38]
Hello David,

apng-js is a great tool for manipulating apng files, thank you for this!

The code has not been updated for 4 years now.
Some forks contain interesting updates or additions (https://github.com/WiseLee0/apng-js-mini) and there are some features that I am about to add too (reverse play using negative values of `playbackRate`).

I don't know what your current situation is but would you agree to give some help by merging interesting forks, including the one I am working on? I think it is always better to contribute to the original project.

Best regards,
Anthony

**Comment by jikkuatwork** [comment]
Indeed, I need reverse play & custom play. It would be great to merge these features.
